README.md

```diff
@@ -581,6 +578,106 @@ This document is written to capture the conventions of writing styles for our pr
 ### Disclaimer
 It's important to note that client and project requirements always undo choices made in these guidelines
 
+### Formatting and Naming
+
+Names should be meaningful and not presentational:
+```css
+/* Bad examples */
+
+/* presentational with color and position reference */
+.button-orange {}
+.left-block {}
+
+/* meaningless */
+.yee-u88 {}
+```
+Some good examples would be:
+```css
+.video {}
+
+.gallery {}
+
+```
+
+Selectors are placed on separate lines:
+```css
+/* Good */
+h1,
+h2,
+h3 {
+  text-transform: uppercase;
+}
+
+/* Bad */
+h1, h2, h3 {
+  text-transform: uppercase;
+}
+```
+
+Don't use ids as a selector (unless there is no other option)
+
+Don't combine element types with classes and ids. Preferably don't style on elements at all. Reasonable exceptions are in a base reset or when styling elements that can not be targeted in another way (e.g. elements that are generated from a wysiwyg editor or coming from external sources).
+```scss
+/* Recommended */
+.overview {}
+
+/* Encouraged exceptions */
+.wysiwyg {
+  h1,
+  h2,
+  h3 {
+    text-transform: uppercase;
+  }
+}
+
+/* Bad */
+ul.overview {}
+
+.overview li {}
+```
+
+Selectors are separated by empty lines
+```css
+/* Good */
+.foo {
+  color: #f00;
+}
+
+.bar {
+  color: #0f0;
+}
+
+.baz {
+  color: #00F;
+}
+
+/* Bad */
+.foo {
+  color: #f00;
+}
+.bar {
+  color: #0f0;
+}
+.baz {
+  color: #00F;
+}
+```
+
+Properties have their own unique line
```

Comment:
Does this imply that a property can't span multiple lines?

Reply:
No, for legibility it would make sense to span multiple lines. i.e. with named grid-templates or combined background styles. I will expand the example to clarify.